context/package.json

```diff
@@ -18,6 +18,7 @@
     "@visx/tooltip": "^1.7.2",
     "ajv": "^6.12.3",
     "bowser": "^2.11.0",
+    "chart.js": "^3.4.1",
```

Comment:
```suggestion
```

Reply:
Alternatively you can just run `npm uninstall chart.js` and it will uninstall the package.